src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -260,6 +247,12 @@ public override int GetHashCode()
             throw new NotSupportedException(SR.NotSupported_CannotCallGetHashCodeOnSpan);
         }
 
+        /// <summary>
+        /// Returns a <see cref="String"/> with the name of the type and the number of elements
+        /// </summary>
+        /// <returns>A <see cref="String"/> with the name of the type and the number of elements</returns>
+        public override string ToString() => $"System.Span[{Length}]";
```

Comment:
Performance here isn't important, but FYI this is more expensive than just:
```C#
"System.Span[" + Length.ToString() + "]";
```
The string interpolation version will box Length, will create an array to store that boxed length, and then will end up calling string.Format, whereas the above allocates the string for length and then calls String.Concat(string, string, string).